salesforce-oms/1.1/modules/ROOT/pages/sfdc-oms-setup-customization-guide.adoc

```diff
@@ -10,12 +10,21 @@ Follow the instructions in the https://resources.docs.salesforce.com/latest/late
 * Disable the *Enable Reduction Orders* option in *Order Settings*.
 * Enable *Order Management*, *Use Person Accounts*, and *High Scale Orders*
 * Activate *Multiple Currencies*
+* Enable *State* and *Country/Territory* picklists
 * <<configure_payment_gateway>>
+* Configure either a digital wallet or credit card payment method and associate an account with it
+* Create a new Digital Wallet from the Digital Wallets page if there is no Payment Method already configured for the Account. A Digital Wallet can be created by filling in the Account, Processing Mode, Nickname and Type fields. 
 
-== Customization
+
+== Customize Salesforce OMS
 
 Install the Salesforce package from the https://anypoint.mulesoft.com/exchange/997d5e99-287f-4f68-bc95-ed435d7c5797/accel-b2b-common-resources-src/[Accelerator B2B Common Resources] project and verify that the configuration is correct in your Salesforce organization.
 
+Follow these steps after installing the package successfully:
+
+* Create matching Rule Account Interchange Code Match with Matching Cretaria Account: B2B_EDISenderCode EXACT MatchBlank = FALSE 
+* Create duplicate rule named 'Account Interchange Code Rule' with the matching rule 'Account Interchange Code Match'. Set the order to 1, giving it precedence over the standard account matching rule.
```

Comment:
```suggestion
* Create a duplicate rule named `Account Interchange Code Rule` in *Setup > Duplicate Rules*. Select `Account Interchange Code Match` as the matching rule. After you save and activate the duplicate rule, open the rule in  *Duplicate Management > Duplicate Rules* and set the order to 1 in the *Reorder* field, which gives the rule precedence over the standard account matching rule.
```

Reply:
@tasmoore We need to add the matching criteria Account: B2B_EDISenderCode EXACT MatchBlank = FALSE